WebViewControl/WebViewLoader.cs

```diff
@@ -39,15 +40,17 @@ internal static class WebViewLoader {
              
             };
 
-            var customSchemes = CustomSchemes.Select(s => new CustomScheme() { SchemeName = s, SchemeHandlerFactory = new SchemeHandlerFactory() }).ToArray();
+            var customSchemes = CustomSchemes.Select(s => new CustomScheme() { 
+                SchemeName = s, 
+                SchemeHandlerFactory = new SchemeHandlerFactory(ownerWebView),
+                IsCorsEnabled = true,
```

Comment:
i dont recall why I needed IsCorsEnabled && IsCSPBypassing

Reply:
Those are related to security restrictions as well

Reply:
yeah ... but try to remove those... and see if it still works

Reply:
Removed the three of them and it still works